src/CommandBinaryBuffer.ts

```diff
@@ -216,6 +271,18 @@ interface IParameterAbsoluteDataStatus {
     status: number
 }
 
+/**
+ * Interval in seconds when device send EXTRA FRAME.
+ * deviceParameters.EXTRA_FRAME_INTERVAL = `28`.
+ */
+interface IParameterExtraFrameInterval {
+    /**
+     * If value = `0` EXTRA FRAME disabled.
```

Comment:
```suggestion
     * If value is `0` EXTRA FRAME disabled.
```
?

as it's not code but sentence

Reply:
fixed